### DISCUSSION TITLE
How to generate highlighted html tags ==this== to <mark>this</mark>

### DISCUSSION THREAD

**Discussion by wchorski** [remarkjs/remark#1066] [General] [answered]
most search results for this question show solutions for "code syntax highlighting".
I want to simply generate <mark> tags around any markdown surrounded by `== ==`

==this==    to    `<mark>this</mark>`

I'm using [react-markdown](https://github.com/remarkjs/react-markdown) to do all my conversion from .md to html. Is there a plugin?

**Comment by ChristianMurphy** [comment, marked as answer]
Commonmark doesn't include this as a feature.
https://commonmark.org/help/
So remark and react-markdown don't include it.

It could be added through a plugin.
There may be an existing one:
https://github.com/remarkjs/remark/blob/main/doc/plugins.md#list-of-plugins
https://github.com/topics/remark-plugin

If there isn't existing one you could create your own. Though I wouldn't necessarily recommend doing so, this page explains more:
https://github.com/micromark/micromark#extending-markdown
